### PR TITLE
Bug 1252235: better handling of shell quoting 

### PR DESCRIPTION
**Pull request by djmitche**
This moves the handling to instantiate, and fixes the error handling for the instantiation.

### CODE REVIEW

File: src/try/instantiate.js [hunk 1]
```diff
@@ -93,6 +95,13 @@ export default function instantiate(template, options) {
     }
   }
 
+  // quote the contained text for passing to the shell; note that this includes
+  // the leading and trailing " or ' characters, so they must not be included
+  // in the template.
+  function shellquote(text, render) {
```
Comment (gregarndt) [inline src/try/instantiate.js]: ah nice putting it here.

File: src/jobs/taskcluster_graph.js [hunk 2]
```diff
@@ -127,7 +119,7 @@ export default class TaskclusterGraphJob extends Base {
     try {
       let graphUrl = projectConfig.tcYamlUrl(this.config.try, urlVariables);
       console.log(`Fetching '.taskcluster.yml' url ${graphUrl} for '${repo.alias}' push id ${push.id}`);
-      let graphText = await fetchGraph(job, graphUrl);
+      let graphText = await fetchGraph(graphUrl);
```
Comment (gregarndt) [inline src/jobs/taskcluster_graph.js]: thanks for fixing this!

**Comment by gregarndt** [comment]
👍

**Comment by djmitche** [comment]
Thanks for deploying it. You had me scared with the email about deploying
changes, since I hadn't reverted master!

Dustin

**Pull request merged**